### DISCUSSION TITLE
Blacklist not Working?

### DISCUSSION THREAD

**Discussion by Originator500** [aclap-dev/video-downloadhelper#268] [Help and Q&A]
### Your question

I've blacklisted Amazon so it doesn't try to download the videos I'm watching. However, even after disabling DASH streaming and blacklisting amazon.com, it still brings up DASH streaming download options.

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): df8fd6afa6c4435e
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\User",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
It's not going to download on it's own unless you click download , plus I believe Amazon uses DRM.
Maybe there are other domains that have to be blacklisted.

**Comment by Originator500** [comment]
> It's not going to download on it's own unless you click download"

oh, okay. I was thinking it was tracking it and taking up bandwidth with whatever it was doing. Ty!

**Comment by DBLClick** [comment]
The one thing I noticed on blacklisting.  It doesn't appear to be a method to block a full domain.  If the domain is abc.com.  then www.abc.com or video.a.abc.com still get's detected.  we need the ability to wildcard a domain to prevent streaming from fully be detected.  

A *domain.com  will get anything in the domain.
